Rubberduck.VBEEditor/Extensions/KeyValuePairExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Rubberduck.VBEditor.Extensions
+{
+    public static class KeyValuePairExtensions
+    {
+        //See https://stackoverflow.com/a/43282724/5536802
+        public static void Deconstruct<TKey, TValue>(this KeyValuePair<TKey, TValue> kvp, out TKey key, out TValue value)
```

Comment:
I know that as destructuring ... probably means the same though...

Reply:
@Vogel612  FWIW `Destructure` sounds very strange, at least in my mind's view. `Deconstruct` is my preference.

Reply:
_Deconstruction_ is the official terminoligy from MS.

Reply:
Disregard then. I wish MS would stop inventing closely resembling, but different names for every little thing somewhen ...